experiments/memcached-cat/main.go

```diff
@@ -158,6 +122,12 @@ func main() {
 	numberOfAvailableCacheWays := uint64(maxCacheWaysToAssign + minCacheWaysToAssign)
 	wholeCacheMask := 1<<numberOfAvailableCacheWays - 1
 
+	if sensitivity.ShouldLaunchKubernetesCluster() {
```

Comment:
@skonefal - I think that launching a cluster should be part of `experiment` rather then `sensitivity package.